.github/workflows/ci.yaml

```diff
@@ -242,56 +259,3 @@ jobs:
         run: |
           echo "Make sure that all sqlc changes are checked in"
           make verify/sqlc
-
-  producer_sample:
```

Comment:
Just to make sure: you took this out on purpose?

Reply:
Yeah, I don’t think it’s needed anymore. We have enough coverage elsewhere imo.